CourseDevelopment/Week5/Chapter5.MD

```diff
@@ -0,0 +1,99 @@
+# Course Development Week 5
+Course Development Week 5 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+
+Date: Dec/07/2022
+
+Name: Hendrik Snijder @hendelhendel
+
+Open tasks @SanliFaez @erikvansebille
++ Feedback on lecture-tutorial distribution of topics
++ Feedback on lecture format
++ Feedback on tutorial format
+
+## Goal: public engagement: story-telling and presentation; activism and trust in scientists
+After this week, one understands the role, responsibilities and interaction of scientists with scientific communities and the general public. 
+Students have to give a presentation about their other assignments using skills learned in the course. 
+Further, the studends are able to formulate their opinion in a discussion on the role a scientist must take in a public debate.
+
+## Keywords student familiarity check week 5
+Some of the keywords in syllabs.MD are supposed to be new, while some are most probably familiar for students.
+
+|**New Concepts**|**Familiar Concepts**|
+|----------------|---------------|
+|How storytelling works|Metaphors|
+|Science communication and journalism|Communicating uncertainty|
```

Comment:
Do you really think communicating uncertainty is familiar? What I mean with it, is how you can communicate that for example quantum computers are available anytime between 2030 and 2050. That is an uncertainty, but the general public may think scientists are simply guessing

Reply:
> What do you think of inviting students from, e.g., economy, history or law-department and let them have a conversation with physics students. In this way the students experience how it feels to bridge between fields.

Haha, that is a cool idea! May be difficult to organise, though (why would students from other fields attend?). But we can at least try something like this?